.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Jira Ticket: 
+
+# Description of Work
+Describe the work that has been done
+    
+# Acceptance Criteria
+- [x] completed acceptance criteria
```

Comment:
Back in the day, I used checkboxes too, but I feel like they only work well if you have some checkboxes that would be left checked versus not checked. Maybe just going as bullets would still convey the criteria is done.

If the pull request is still in draft, I would not expect all the criteria to be done yet.

Reply:
Changed from checkboxes to bullets